src/Illuminate/Queue/Connectors/BeanstalkdConnector.php

```diff
@@ -32,11 +32,10 @@ public function connect(array $config)
      */
     protected function pheanstalk(array $config)
     {
-        return new Pheanstalk(
+        return Pheanstalk::connect(
```

Comment:
@crynobone I've been looking at the code here and I think something's off. There doesn't seems to be a `connect` method in v4. You've added the `create` method which is the correct one I believe in https://github.com/laravel/framework/pull/27622

Reply:
If you fix this can you perhaps send in a test?

Reply:
Sorry, yes `create()` is the correct method. 

To add a test for this we need to include beanstalk on travis, okay on that?

Reply:
No that's ok. I've pushed a commit with the correct method. Thanks!